setup/src/Magento/Setup/Model/Installer.php

```diff
@@ -1157,6 +1185,15 @@ public function cleanupDb()
     }
 
     /**
+     * getObjectManager
+     * @return \Magento\Framework\ObjectManagerInterface
+     */
+    public function getObjectManager()
```

Comment:
why do we need this method in the interface? Please, remove this method. This interface should not provide such methods